### DISCUSSION TITLE
How to stop posters overwriting?

### DISCUSSION THREAD

**Discussion by pablomcnally** [meisnate12/Plex-Meta-Manager#41] [Q&A] [answered]
Hiya guys! I finally got this working more or less after a few traumas, thanks for all the brilliant work. Still having a bit of trouble here and there with the YML but, in the main, it's up and running.

Quick question which I'm sure is something obv. If I set up a collection for an actor - say Clint Eastwood, it pulls the collection together and adds a photo of the actor as the poster from TMDB. I changed that out to one that matches the style of the rest of my collections, but when I run the script again it overwrites my poster and replaces it with the one from TMDB. How do I turn that off :-)

It only does it on actors, the ones I have set for Trending movies etc maintain my poster.

**Comment by meisnate12** [comment, marked as answer]
im assuming your doing something like this for you actor collections
```yaml
collections:
  Robin Williams:
    actor: tmdb
    tmdb_person: 2157
```

tmdb_person will update the summary and profile of the collection to the tmdb person to have it only update the bio youd do something like this

```yaml
collections:
  Robin Williams:
    actor: Robin Williams
    tmdb_biography: 2157
```

or you could use the assets directory (this is what i do). You set the asset directory location via the plex attribute by default its located in the same directory as your config.yml at `/config/assets`. Then the program will automatically look in the assets folder for a folder with a name corresponding to the collection name and then looks inside that folder for a file called poster and a file called background to update your collection images. For the example above the folder would be called Robin Williams  i.e.:

```
├── config
│   ├── config.yml
│   ├── Movies.yml
│   ├── assets
│       ├── Robin Williams
│           ├── background.png
│           ├── poster.png
```

Having a local asset will override tmdb grabbing the poster because it already has one.

**Reply by pablomcnally** [reply]
Brilliant stuff. Thanks pal!
